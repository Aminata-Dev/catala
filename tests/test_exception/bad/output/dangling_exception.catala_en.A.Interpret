[ERROR] Unknown label for the scope variable x: "base_y"

  --> tests/test_exception/bad/dangling_exception.catala_en
   | 
12 |   exception base_y
   |             ^^^^^^
   + Test
