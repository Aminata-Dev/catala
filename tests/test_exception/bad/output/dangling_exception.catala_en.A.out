[ERROR] Unknown label for variable x: "base_y"

  --> test_exception/bad/dangling_exception.catala_en
   | 
12 |   exception base_y
   |             ^^^^^^
   + Test
